New Features

RishiAstra opened the pull request:
This pull request is composed of new features that I have added with the advice of Mr. Goddard. New features include:
- Fossil fuels added (methane-octane)
- Spawn Balls that spawn sets of molecules (and delete old molecules) when touched to the trigger Spawn Ball
- Dino (a dinosaur) who lives on the Chicxulub crater. Dino can be picked up and shaken to produce decaying dinosaurs. Decaying dinosaurs produce random fossil fuels after turning black.
- Sounds added for picking up Dino, spawning decaying dinosaurs, and fossil fuel names
- Slight reorganization of folders (e.g. Assets/Scripts/Tools has been split into two folders: Hand and Utility)
- Thicker box colliders around the molecule area, preventing molecules from escaping to the void and beyond
- Slight changes to existing scripts including sending OnTractor, OnGrab, OnRelease messages to objects grabbed by hand or tractored

RishiAstra committed:
attempted to open in Unity 2019

RishiAstra committed:
Revert "attempted to open in Unity 2019"

This reverts commit 272d0ac7745d42b8e6fcc44cd5b69ac52cdf365c.

RishiAstra committed:
Added ethane

RishiAstra committed:
Added fossil fuel spawning

RishiAstra committed:
Finished (hopefully) Spawn Balls

Spawn balls spawn molecules when touched to the spawn ball trigger. This is useful to load different sets of molecules.

RishiAstra committed:
Added audio on spawning and built the project

RishiAstra committed:
adding dino

RishiAstra committed:
Added sounds and fixed sounds of molecules spawned by spawn balls

RishiAstra committed:
saved scene changes

RishiAstra committed:
dino added and 2 name audios

RishiAstra committed:
In progress (please disregard)

Please disregard this commit. In order to work between two devices, I have to commit even though this is still work in progress.

RishiAstra committed:
Dino audio

For some reason, the dino audio didn't get pushed in the last commit.

RishiAstra committed:
code cleanup (in progress)

Added comments and tooltips, cleaned up code to be more neat

RishiAstra committed:
Fixed audio glitch

Audio would not play when some molecules were selected

tomgoddard commented:
Hi Rishi,

  Thanks for making the pull request.  This may take some time, a few weeks or few months for us to look over and test.  Alan and Ray are busy with college classes and I am busy with work.  Be patient and we will get to it.  Very much appreciate your effort on MolZoo!

  A key part of science work is to tell people what you did.  That motivates them to do something even better, they can see your ideas and don't have to start from scratch.  For MolZoo the clearest way to do that is to make a video showing the cool features, ideally with some audio narration explaining what is going on.  If you are interested in making such a video that could be posted on YouTube or Twitter that will help people discover and appreciate your work.  If you are tired of MolZoo that is ok too.

	Tom


> On May 19, 2020, at 5:13 PM, RishiAstra <notifications@github.com> wrote:
> 
> This pull request is composed of new features that I have added with the advice of Mr. Goddard. New features include:
> 
> Fossil fuels added (methane-octane)
> Spawn Balls that spawn sets of molecules (and delete old molecules) when touched to the trigger Spawn Ball
> Dino (a dinosaur) who lives on the Chicxulub crater. Dino can be picked up and shaken to produce decaying dinosaurs. Decaying dinosaurs produce random fossil fuels after turning black.
> Sounds added for picking up Dino, spawning decaying dinosaurs, and fossil fuel names
> Slight reorganization of folders (e.g. Assets/Scripts/Tools has been split into two folders: Hand and Utility)
> Thicker box colliders around the molecule area, preventing molecules from escaping to the void and beyond
> Slight changes to existing scripts including sending OnTractor, OnGrab, OnRelease messages to objects grabbed by hand or tractored
> You can view, comment on, or merge this pull request online at:
> 
>   https://github.com/alanbrilliant/MolecularZoo/pull/1 <https://github.com/alanbrilliant/MolecularZoo/pull/1>
> Commit Summary
> 
> attempted to open in Unity 2019
> Revert "attempted to open in Unity 2019"
> Added ethane
> Added fossil fuel spawning
> Finished (hopefully) Spawn Balls
> Added audio on spawning and built the project
> adding dino
> Added sounds and fixed sounds of molecules spawned by spawn balls
> saved scene changes
> dino added and 2 name audios
> In progress (please disregard)
> Dino audio
> code cleanup (in progress)
> Fixed audio glitch
> File Changes
> 
> A Assets/Dino.meta <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-7fd809aa691b2c3d3731bd4cbb18c449> (8)
> A Assets/Dino/Model.meta <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-dad7ce4615e84404c1303a955462e7d4> (8)
> A Assets/Dino/Model/Comp.fbx <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-f2b25c37090b6c801774bb0ae4c0d23d> (0)
> A Assets/Dino/Model/Comp.fbx.meta <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-136d1b9e77f841da12e9a64358ac382b> (5596)
> A Assets/Dino/Model/Materials.meta <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-2cb41a0ebee7524d570cdd08ca87a577> (8)
> A Assets/Dino/Model/Materials/comp.mat <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-1780507d55f77fffc849a6c507348cf8> (76)
> A Assets/Dino/Model/Materials/comp.mat.meta <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-9e206a2660d3c4342dde80f748d352fd> (8)
> A Assets/Dino/Model/Materials/compA.png <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-6e9ce709ef26e8156bffcca4cac81292> (0)
> A Assets/Dino/Model/Materials/compA.png.meta <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-f776de571871cbfec4a1474a26bd6480> (92)
> A Assets/Dino/Model/Materials/compB.png <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-c0c91a48f5622d02755d5abef2d64167> (0)
> A Assets/Dino/Model/Materials/compB.png.meta <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-6a90802e7437059ffb37fcc8c3fdbf07> (92)
> A Assets/Dino/Model/Materials/compC.png <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-040bcc26ffb5e0b771fdd25b55c271bd> (0)
> A Assets/Dino/Model/Materials/compC.png.meta <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-86820e03fd672654954c230b0b296dbd> (92)
> A Assets/Dino/Model/Materials/compN.png <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-4da251bf657f381e66e12e5e676417e1> (0)
> A Assets/Dino/Model/Materials/compN.png.meta <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-5eb8e55c324a28f3a45c46b8545cb8dd> (84)
> A Assets/Dino/Model/Materials/eye normal.png <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-3be6a4bacc669635ce1ec358222c8df3> (0)
> A Assets/Dino/Model/Materials/eye normal.png.meta <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-0299141cebf4d9929e23927ab1e66b10> (99)
> A Assets/Dino/Model/Materials/eye.mat <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-ba178f2bd6dc1bccb6f0ba1370b7b281> (76)
> A Assets/Dino/Model/Materials/eye.mat.meta <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-e8df2ccc499e1b95ee1b9b0836ee0392> (8)
> A Assets/Dino/Model/Materials/eye.png <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-2417ab0e471d157889eccebc5b2b49f0> (0)
> A Assets/Dino/Model/Materials/eye.png.meta <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-1dd40974cf6f18596f28d468ec7f6be7> (88)
> A Assets/Dino/Model/Materials/eye.psd <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-6f5a4a57acd1a85f9c982b02d5d9c694> (0)
> A Assets/Dino/Model/Materials/eye.psd.meta <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-01190a86ee4bb7a815a4e86778740e4c> (78)
> A Assets/Dino/Model/comp.controller <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-54f8a8ad9e187271915ca6f5c8ba663e> (3135)
> A Assets/Dino/Model/comp.controller.meta <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-5408901a698705fc33bf593ee5af2213> (8)
> A Assets/Dino/Prefabs.meta <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-14806cbaf7aaa6dea50df7be2ba91fab> (8)
> A Assets/Dino/Prefabs/Compsognathus.prefab <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-e05ce69fe17b20291139e25f75cff58b> (2520)
> A Assets/Dino/Prefabs/Compsognathus.prefab.meta <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-6a5039cf8928560e503cde0d2aeaa541> (9)
> A Assets/Dino/Prefabs/Dino.prefab <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-0395005e4aa08776f6b878d841743566> (2308)
> A Assets/Dino/Prefabs/Dino.prefab.meta <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-2685e38cde803f055afd402aea926740> (8)
> A Assets/Dino/Prefabs/DinoDecay.prefab <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-11bf5a68fe5eabb37a0d331f636ec4b4> (2265)
> A Assets/Dino/Prefabs/DinoDecay.prefab.meta <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-e4fbaff3a7f1d0a358d415c22bb40ccc> (8)
> A Assets/Materials/Spawn Balls.meta <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-6643158ee667875827af032d353a2814> (8)
> A Assets/Materials/Spawn Balls/Spawn Ball Default.mat <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-f7a17fc12bdec380dab3c9524b31617a> (76)
> A Assets/Materials/Spawn Balls/Spawn Ball Default.mat.meta <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-57c57bf943ffa1986e8ddce22e08ed26> (8)
> A Assets/Materials/Spawn Balls/Spawn Ball Fossil.mat <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-dc00345d1dd212d9965f42d3d00f5156> (76)
> A Assets/Materials/Spawn Balls/Spawn Ball Fossil.mat.meta <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-b2cebd8bac22e2db7ac00abe9e3402e6> (8)
> A Assets/Materials/Spawn Balls/Spawn Ball Trigger.mat <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-ad637ccd074031593e1c38c090d8f3fe> (76)
> A Assets/Materials/Spawn Balls/Spawn Ball Trigger.mat.meta <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-1a58af177a6ef6298538150cac13dfa8> (8)
> A Assets/Materials/Spawn Balls/Spawn Ball default paricles.mat <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-d90b3d38a59b04883bb9eb2c6b7be2a5> (78)
> A Assets/Materials/Spawn Balls/Spawn Ball default paricles.mat.meta <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-b27e8398ce4b90766ee68ad996c61b64> (8)
> A Assets/Materials/Spawn Balls/Spawn Ball fossil particles.mat <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-6ef253aa972ad915a20d94b97e42fbcf> (78)
> A Assets/Materials/Spawn Balls/Spawn Ball fossil particles.mat.meta <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-b2c865085a06f1700a8a029a1c8b59cf> (8)
> A Assets/Materials/Spawn Balls/Spawn Ball trigger particles.mat <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-992fe4084900fc5fdf09bc1ebdd3a7a6> (78)
> A Assets/Materials/Spawn Balls/Spawn Ball trigger particles.mat.meta <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-4d468deb4ef82f9ae7287ece7fbd1bbb> (8)
> M Assets/Scenes/Molecules.unity <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-3c24ab63ba3157adba586bcad19bb693> (16512)
> M Assets/Scripts/Atomic Scripts/AtomScript.cs <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-84dca89ad41ad0aa15fb44cec365e9ee> (8)
> M Assets/Scripts/Data/DataManager.cs <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-1346bca5ec49d4ef274cbaff9129d7f1> (3)
> A Assets/Scripts/Dino.meta <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-dad993486c3218d751145ac130af8d3b> (8)
> A Assets/Scripts/Dino/Dino.cs <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-7abaae6a4ce950263a5217c8375a9512> (56)
> A Assets/Scripts/Dino/Dino.cs.meta <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-431fe3b5810214f49bdfbff0d216276c> (11)
> M Assets/Scripts/MoleculeCreator.cs <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-337134538627a63dd94a476af75179d7> (28)
> A Assets/Scripts/Tools/Hand.meta <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-afe81f7c9f72eca4650e6ac2d4ed9037> (8)
> R Assets/Scripts/Tools/Hand/CardDeck.cs <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-29c37b472ee51beac6057f4d1a13b929> (0)
> R Assets/Scripts/Tools/Hand/CardDeck.cs.meta <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-5e1da8745d8573cbd5da38f096f501e8> (0)
> R Assets/Scripts/Tools/Hand/Gun.cs <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-f1ebe9181546406f9659e55d94f3bf33> (0)
> R Assets/Scripts/Tools/Hand/Gun.cs.meta <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-5114d0a7c8a410de193290e992bb76c7> (0)
> R Assets/Scripts/Tools/Hand/HeavyPistol.cs <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-cc26c9052b39e15f109f7578773156dd> (0)
> R Assets/Scripts/Tools/Hand/HeavyPistol.cs.meta <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-e76bf56176d6271180a8530cda41fac9> (0)
> R Assets/Scripts/Tools/Hand/Pistol.cs <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-484404d721c2937bf5b1f258c3ee53ef> (0)
> R Assets/Scripts/Tools/Hand/Pistol.cs.meta <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-8d543adf2231ca16616b57b64d6f653e> (0)
> R Assets/Scripts/Tools/Hand/Tractor.cs <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-38c8576fffaf51d5361bc4016508ca94> (12)
> R Assets/Scripts/Tools/Hand/Tractor.cs.meta <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-b0dd02d605d3839073a3af8a5ff81a67> (0)
> A Assets/Scripts/Tools/Utility.meta <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-cc6d8e2af24bf7c8579c92c1e5dcaf9e> (8)
> A Assets/Scripts/Tools/Utility/Decay.cs <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-d2dc156e803e0130a4f4ef789bcd8f1f> (91)
> A Assets/Scripts/Tools/Utility/Decay.cs.meta <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-64137228bc07c610cee1c6780ce3493a> (11)
> A Assets/Scripts/Tools/Utility/FollowWithForce.cs <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-e1d198deef38e9be5af8be9aaf163edb> (32)
> A Assets/Scripts/Tools/Utility/FollowWithForce.cs.meta <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-3cbf94604ad1ed72cc0be8ddadc992cd> (11)
> A Assets/Scripts/Tools/Utility/GrabbableCollider.cs <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-4b50a3a11fd5132d8c5aa8fe2ca68931> (13)
> A Assets/Scripts/Tools/Utility/GrabbableCollider.cs.meta <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-9416b37c31770b738492e5ba9d094cd7> (11)
> A Assets/Scripts/Tools/Utility/SetAnimatorParameter.cs <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-d5ff33d899690789ea73f2708b6825f9> (88)
> A Assets/Scripts/Tools/Utility/SetAnimatorParameter.cs.meta <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-48db24a79f00c2f7b2b4c3aa257c8360> (11)
> A Assets/Scripts/Tools/Utility/SpawnOnContact.cs <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-c89da41e7a4f25fe51d776019988d4c9> (162)
> A Assets/Scripts/Tools/Utility/SpawnOnContact.cs.meta <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-1d27ead4b14b252af3c7311de5fa5f54> (11)
> A Assets/Scripts/Tools/Utility/SpawnOnShake.cs <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-62c20331226d8d618e6e1c350c0ff9d8> (65)
> A Assets/Scripts/Tools/Utility/SpawnOnShake.cs.meta <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-a45f9d987b2c688a2a343bec62e2e21e> (11)
> M Assets/Scripts/Wand.cs <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-88f88828271962312568f3c1b4d9d520> (11)
> A Assets/Sound/ButaneSound.mp3 <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-b0d64686d838685499b7257a3944b9a0> (0)
> A Assets/Sound/ButaneSound.mp3.meta <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-371837a1712eeefe10e6e279717bf9b5> (22)
> A Assets/Sound/DefaultSpawn.mp3 <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-96d6ad3b47fb7c523ae9bc02df1c6a87> (0)
> A Assets/Sound/DefaultSpawn.mp3.meta <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-d8b45eeac3c5ee261e9fff5c0e9060c8> (22)
> A Assets/Sound/EthaneSound.mp3 <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-ef16bb932a5df900633bbcff0a41da83> (0)
> A Assets/Sound/EthaneSound.mp3.meta <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-2c9eb10c2236ab326418e406a8e31c05> (22)
> A Assets/Sound/FossilFuelsSpawn.mp3 <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-4d683439d0eeb9ffce0942e020155ca0> (0)
> A Assets/Sound/FossilFuelsSpawn.mp3.meta <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-6521be085efe66b2dcbfdf89a9418e12> (22)
> A Assets/Sound/HeptaneSound.mp3 <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-0c6557a9cecbfafbfff100e0f92d9774> (0)
> A Assets/Sound/HeptaneSound.mp3.meta <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-1ff05c5f9e1886eda4650ad58f6fafbe> (22)
> A Assets/Sound/HexaneSound.mp3 <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-6c2cd6893514ea2f5717c145a1a68e5b> (0)
> A Assets/Sound/HexaneSound.mp3.meta <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-31bc6e37c809ee89cfbe86f25e69056b> (22)
> A Assets/Sound/MethaneSound.mp3 <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-0223dee25d6165f500e70e12fa67a3f2> (0)
> A Assets/Sound/MethaneSound.mp3.meta <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-0b29146abed75af1f662e6861ab6018a> (22)
> A Assets/Sound/OctaneSound.mp3 <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-34b78f107b216ff1de224d43008f05d3> (0)
> A Assets/Sound/OctaneSound.mp3.meta <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-ec9370c239ec4eb170a08a5adcd3e884> (22)
> A Assets/Sound/PentaneSound.mp3 <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-ba09042521c6781facc5d912e94b85e6> (0)
> A Assets/Sound/PentaneSound.mp3.meta <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-7907e1ab653db6c511cd88189e50c726> (22)
> A Assets/Sound/PropaneSound.mp3 <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-7d6111a8a60284027fc613b0267f0f63> (0)
> A Assets/Sound/PropaneSound.mp3.meta <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-c4c7c53a0556001dafb7cf3b71159585> (22)
> A Assets/Sound/dino roar.mp3 <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-255122c139b6a7345ad8d642e4f21a1b> (0)
> A Assets/Sound/dino roar.mp3.meta <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-63b623d5f1fb803ac92e169ec8ba04c2> (22)
> A Assets/Sound/dino spawn.mp3 <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-7c9b24da97816bc0d91fa2cad7e673ec> (0)
> A Assets/Sound/dino spawn.mp3.meta <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-87fb9d294091b3c16bb9bf3600e81e8a> (22)
> A Assets/StreamingAssets/butanedata.json <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-7bea4901955193c35826abb424387b2f> (381)
> A Assets/StreamingAssets/butanedata.json.meta <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-05083d4f0ffcea8f62f4d3b0144dd3b9> (7)
> A Assets/StreamingAssets/ethanedata.json <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-9ae8ddb1249103970dde8b27dfb6fd0e> (322)
> A Assets/StreamingAssets/ethanedata.json.meta <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-0ab4282bb833d242976e42e153bec18e> (7)
> A Assets/StreamingAssets/heptanedata.json <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-15d06778f0b598cc782995f40fa6a995> (478)
> A Assets/StreamingAssets/heptanedata.json.meta <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-7db429da67078414e5861836624c82f7> (7)
> A Assets/StreamingAssets/hexanedata.json <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-0b21327cdf2e8d170e19747ee0a21e2f> (447)
> A Assets/StreamingAssets/hexanedata.json.meta <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-a46d9f4a5e7da93630c79e7c628555b1> (7)
> A Assets/StreamingAssets/methanedata.json <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-b47df1ac06c485a3f81e57ba7b1c3c56> (292)
> A Assets/StreamingAssets/methanedata.json.meta <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-4a1e0f679b4c2a46d86c7091d9832272> (7)
> A Assets/StreamingAssets/octanedata.json <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-89440417d9812c590b654a40dc460d9c> (537)
> A Assets/StreamingAssets/octanedata.json.meta <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-540cf859493339be78c19a62276de0b6> (7)
> A Assets/StreamingAssets/pentanedata.json <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-256cc8ac7939e99004701c9c9a65d249> (411)
> A Assets/StreamingAssets/pentanedata.json.meta <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-4473d7103e9ed742e84af13376cf1418> (7)
> A Assets/StreamingAssets/propanedata.json <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-cda1ffb2e2ac1694c654bc59a8cb306f> (348)
> A Assets/StreamingAssets/propanedata.json.meta <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-39f7c527d5cf469119d428b65c021d9f> (7)
> M ProjectSettings/EditorBuildSettings.asset <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-8f3c83eb79d0b4aa8cf581a168df971c> (18)
> M ProjectSettings/ProjectSettings.asset <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-988fde71e6a4c917d4d31be70b435aa1> (6)
> M ProjectSettings/TagManager.asset <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-c100fe01e4868a34c50c3f040849a2c6> (1)
> M ProjectSettings/UnityConnectSettings.asset <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-3e2b55b7cb4b6b167c1e81679c6d94c9> (5)
> A Rishi Astra - MolZooBuild.zip <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-fd4683e637696e8525e714b007172d4c> (0)
> A build.zip <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-bd3f409385be2426c686b5e8eb288c66> (0)
> M obj/Debug/Assembly-CSharp.csproj.FileListAbsolute.txt <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-84f93d4b7424e81423e9027b246070e4> (169)
> M obj/Debug/Assembly-CSharp.dll <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-8f75c790a28d386e4316fcc8fb50be98> (0)
> M obj/Debug/Assembly-CSharp.pdb <https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-015c7d7ef0edc0f82de440447fb61084> (0)
> Patch Links:
> 
> https://github.com/alanbrilliant/MolecularZoo/pull/1.patch <https://github.com/alanbrilliant/MolecularZoo/pull/1.patch>
> https://github.com/alanbrilliant/MolecularZoo/pull/1.diff <https://github.com/alanbrilliant/MolecularZoo/pull/1.diff>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub <https://github.com/alanbrilliant/MolecularZoo/pull/1>, or unsubscribe <https://github.com/notifications/unsubscribe-auth/AB6JDDJ6VSO64KONV5BYGOTRSMOC3ANCNFSM4NFOUAQA>.
>

missingno01 commented:
Hi Rishi, Tom,

Looks like some awesome additions! I’m just wrapping up the semester here,
so I’ll hopefully aim to run some tests next week(assuming my home desktop
runs MolZoo well enough).

All the best,
Ray

On Tue, May 19, 2020 at 5:37 PM tomgoddard <notifications@github.com> wrote:

> Hi Rishi,
>
> Thanks for making the pull request. This may take some time, a few weeks
> or few months for us to look over and test. Alan and Ray are busy with
> college classes and I am busy with work. Be patient and we will get to it.
> Very much appreciate your effort on MolZoo!
>
> A key part of science work is to tell people what you did. That motivates
> them to do something even better, they can see your ideas and don't have to
> start from scratch. For MolZoo the clearest way to do that is to make a
> video showing the cool features, ideally with some audio narration
> explaining what is going on. If you are interested in making such a video
> that could be posted on YouTube or Twitter that will help people discover
> and appreciate your work. If you are tired of MolZoo that is ok too.
>
> Tom
>
>
> > On May 19, 2020, at 5:13 PM, RishiAstra <notifications@github.com>
> wrote:
> >
> > This pull request is composed of new features that I have added with the
> advice of Mr. Goddard. New features include:
> >
> > Fossil fuels added (methane-octane)
> > Spawn Balls that spawn sets of molecules (and delete old molecules) when
> touched to the trigger Spawn Ball
> > Dino (a dinosaur) who lives on the Chicxulub crater. Dino can be picked
> up and shaken to produce decaying dinosaurs. Decaying dinosaurs produce
> random fossil fuels after turning black.
> > Sounds added for picking up Dino, spawning decaying dinosaurs, and
> fossil fuel names
> > Slight reorganization of folders (e.g. Assets/Scripts/Tools has been
> split into two folders: Hand and Utility)
> > Thicker box colliders around the molecule area, preventing molecules
> from escaping to the void and beyond
> > Slight changes to existing scripts including sending OnTractor, OnGrab,
> OnRelease messages to objects grabbed by hand or tractored
> > You can view, comment on, or merge this pull request online at:
> >
> > https://github.com/alanbrilliant/MolecularZoo/pull/1 <
> https://github.com/alanbrilliant/MolecularZoo/pull/1>
> > Commit Summary
> >
> > attempted to open in Unity 2019
> > Revert "attempted to open in Unity 2019"
> > Added ethane
> > Added fossil fuel spawning
> > Finished (hopefully) Spawn Balls
> > Added audio on spawning and built the project
> > adding dino
> > Added sounds and fixed sounds of molecules spawned by spawn balls
> > saved scene changes
> > dino added and 2 name audios
> > In progress (please disregard)
> > Dino audio
> > code cleanup (in progress)
> > Fixed audio glitch
> > File Changes
> >
> > A Assets/Dino.meta <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-7fd809aa691b2c3d3731bd4cbb18c449>
> (8)
> > A Assets/Dino/Model.meta <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-dad7ce4615e84404c1303a955462e7d4>
> (8)
> > A Assets/Dino/Model/Comp.fbx <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-f2b25c37090b6c801774bb0ae4c0d23d>
> (0)
> > A Assets/Dino/Model/Comp.fbx.meta <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-136d1b9e77f841da12e9a64358ac382b>
> (5596)
> > A Assets/Dino/Model/Materials.meta <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-2cb41a0ebee7524d570cdd08ca87a577>
> (8)
> > A Assets/Dino/Model/Materials/comp.mat <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-1780507d55f77fffc849a6c507348cf8>
> (76)
> > A Assets/Dino/Model/Materials/comp.mat.meta <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-9e206a2660d3c4342dde80f748d352fd>
> (8)
> > A Assets/Dino/Model/Materials/compA.png <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-6e9ce709ef26e8156bffcca4cac81292>
> (0)
> > A Assets/Dino/Model/Materials/compA.png.meta <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-f776de571871cbfec4a1474a26bd6480>
> (92)
> > A Assets/Dino/Model/Materials/compB.png <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-c0c91a48f5622d02755d5abef2d64167>
> (0)
> > A Assets/Dino/Model/Materials/compB.png.meta <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-6a90802e7437059ffb37fcc8c3fdbf07>
> (92)
> > A Assets/Dino/Model/Materials/compC.png <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-040bcc26ffb5e0b771fdd25b55c271bd>
> (0)
> > A Assets/Dino/Model/Materials/compC.png.meta <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-86820e03fd672654954c230b0b296dbd>
> (92)
> > A Assets/Dino/Model/Materials/compN.png <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-4da251bf657f381e66e12e5e676417e1>
> (0)
> > A Assets/Dino/Model/Materials/compN.png.meta <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-5eb8e55c324a28f3a45c46b8545cb8dd>
> (84)
> > A Assets/Dino/Model/Materials/eye normal.png <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-3be6a4bacc669635ce1ec358222c8df3>
> (0)
> > A Assets/Dino/Model/Materials/eye normal.png.meta <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-0299141cebf4d9929e23927ab1e66b10>
> (99)
> > A Assets/Dino/Model/Materials/eye.mat <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-ba178f2bd6dc1bccb6f0ba1370b7b281>
> (76)
> > A Assets/Dino/Model/Materials/eye.mat.meta <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-e8df2ccc499e1b95ee1b9b0836ee0392>
> (8)
> > A Assets/Dino/Model/Materials/eye.png <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-2417ab0e471d157889eccebc5b2b49f0>
> (0)
> > A Assets/Dino/Model/Materials/eye.png.meta <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-1dd40974cf6f18596f28d468ec7f6be7>
> (88)
> > A Assets/Dino/Model/Materials/eye.psd <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-6f5a4a57acd1a85f9c982b02d5d9c694>
> (0)
> > A Assets/Dino/Model/Materials/eye.psd.meta <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-01190a86ee4bb7a815a4e86778740e4c>
> (78)
> > A Assets/Dino/Model/comp.controller <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-54f8a8ad9e187271915ca6f5c8ba663e>
> (3135)
> > A Assets/Dino/Model/comp.controller.meta <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-5408901a698705fc33bf593ee5af2213>
> (8)
> > A Assets/Dino/Prefabs.meta <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-14806cbaf7aaa6dea50df7be2ba91fab>
> (8)
> > A Assets/Dino/Prefabs/Compsognathus.prefab <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-e05ce69fe17b20291139e25f75cff58b>
> (2520)
> > A Assets/Dino/Prefabs/Compsognathus.prefab.meta <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-6a5039cf8928560e503cde0d2aeaa541>
> (9)
> > A Assets/Dino/Prefabs/Dino.prefab <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-0395005e4aa08776f6b878d841743566>
> (2308)
> > A Assets/Dino/Prefabs/Dino.prefab.meta <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-2685e38cde803f055afd402aea926740>
> (8)
> > A Assets/Dino/Prefabs/DinoDecay.prefab <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-11bf5a68fe5eabb37a0d331f636ec4b4>
> (2265)
> > A Assets/Dino/Prefabs/DinoDecay.prefab.meta <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-e4fbaff3a7f1d0a358d415c22bb40ccc>
> (8)
> > A Assets/Materials/Spawn Balls.meta <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-6643158ee667875827af032d353a2814>
> (8)
> > A Assets/Materials/Spawn Balls/Spawn Ball Default.mat <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-f7a17fc12bdec380dab3c9524b31617a>
> (76)
> > A Assets/Materials/Spawn Balls/Spawn Ball Default.mat.meta <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-57c57bf943ffa1986e8ddce22e08ed26>
> (8)
> > A Assets/Materials/Spawn Balls/Spawn Ball Fossil.mat <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-dc00345d1dd212d9965f42d3d00f5156>
> (76)
> > A Assets/Materials/Spawn Balls/Spawn Ball Fossil.mat.meta <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-b2cebd8bac22e2db7ac00abe9e3402e6>
> (8)
> > A Assets/Materials/Spawn Balls/Spawn Ball Trigger.mat <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-ad637ccd074031593e1c38c090d8f3fe>
> (76)
> > A Assets/Materials/Spawn Balls/Spawn Ball Trigger.mat.meta <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-1a58af177a6ef6298538150cac13dfa8>
> (8)
> > A Assets/Materials/Spawn Balls/Spawn Ball default paricles.mat <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-d90b3d38a59b04883bb9eb2c6b7be2a5>
> (78)
> > A Assets/Materials/Spawn Balls/Spawn Ball default paricles.mat.meta <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-b27e8398ce4b90766ee68ad996c61b64>
> (8)
> > A Assets/Materials/Spawn Balls/Spawn Ball fossil particles.mat <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-6ef253aa972ad915a20d94b97e42fbcf>
> (78)
> > A Assets/Materials/Spawn Balls/Spawn Ball fossil particles.mat.meta <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-b2c865085a06f1700a8a029a1c8b59cf>
> (8)
> > A Assets/Materials/Spawn Balls/Spawn Ball trigger particles.mat <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-992fe4084900fc5fdf09bc1ebdd3a7a6>
> (78)
> > A Assets/Materials/Spawn Balls/Spawn Ball trigger particles.mat.meta <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-4d468deb4ef82f9ae7287ece7fbd1bbb>
> (8)
> > M Assets/Scenes/Molecules.unity <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-3c24ab63ba3157adba586bcad19bb693>
> (16512)
> > M Assets/Scripts/Atomic Scripts/AtomScript.cs <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-84dca89ad41ad0aa15fb44cec365e9ee>
> (8)
> > M Assets/Scripts/Data/DataManager.cs <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-1346bca5ec49d4ef274cbaff9129d7f1>
> (3)
> > A Assets/Scripts/Dino.meta <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-dad993486c3218d751145ac130af8d3b>
> (8)
> > A Assets/Scripts/Dino/Dino.cs <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-7abaae6a4ce950263a5217c8375a9512>
> (56)
> > A Assets/Scripts/Dino/Dino.cs.meta <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-431fe3b5810214f49bdfbff0d216276c>
> (11)
> > M Assets/Scripts/MoleculeCreator.cs <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-337134538627a63dd94a476af75179d7>
> (28)
> > A Assets/Scripts/Tools/Hand.meta <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-afe81f7c9f72eca4650e6ac2d4ed9037>
> (8)
> > R Assets/Scripts/Tools/Hand/CardDeck.cs <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-29c37b472ee51beac6057f4d1a13b929>
> (0)
> > R Assets/Scripts/Tools/Hand/CardDeck.cs.meta <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-5e1da8745d8573cbd5da38f096f501e8>
> (0)
> > R Assets/Scripts/Tools/Hand/Gun.cs <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-f1ebe9181546406f9659e55d94f3bf33>
> (0)
> > R Assets/Scripts/Tools/Hand/Gun.cs.meta <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-5114d0a7c8a410de193290e992bb76c7>
> (0)
> > R Assets/Scripts/Tools/Hand/HeavyPistol.cs <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-cc26c9052b39e15f109f7578773156dd>
> (0)
> > R Assets/Scripts/Tools/Hand/HeavyPistol.cs.meta <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-e76bf56176d6271180a8530cda41fac9>
> (0)
> > R Assets/Scripts/Tools/Hand/Pistol.cs <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-484404d721c2937bf5b1f258c3ee53ef>
> (0)
> > R Assets/Scripts/Tools/Hand/Pistol.cs.meta <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-8d543adf2231ca16616b57b64d6f653e>
> (0)
> > R Assets/Scripts/Tools/Hand/Tractor.cs <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-38c8576fffaf51d5361bc4016508ca94>
> (12)
> > R Assets/Scripts/Tools/Hand/Tractor.cs.meta <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-b0dd02d605d3839073a3af8a5ff81a67>
> (0)
> > A Assets/Scripts/Tools/Utility.meta <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-cc6d8e2af24bf7c8579c92c1e5dcaf9e>
> (8)
> > A Assets/Scripts/Tools/Utility/Decay.cs <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-d2dc156e803e0130a4f4ef789bcd8f1f>
> (91)
> > A Assets/Scripts/Tools/Utility/Decay.cs.meta <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-64137228bc07c610cee1c6780ce3493a>
> (11)
> > A Assets/Scripts/Tools/Utility/FollowWithForce.cs <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-e1d198deef38e9be5af8be9aaf163edb>
> (32)
> > A Assets/Scripts/Tools/Utility/FollowWithForce.cs.meta <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-3cbf94604ad1ed72cc0be8ddadc992cd>
> (11)
> > A Assets/Scripts/Tools/Utility/GrabbableCollider.cs <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-4b50a3a11fd5132d8c5aa8fe2ca68931>
> (13)
> > A Assets/Scripts/Tools/Utility/GrabbableCollider.cs.meta <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-9416b37c31770b738492e5ba9d094cd7>
> (11)
> > A Assets/Scripts/Tools/Utility/SetAnimatorParameter.cs <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-d5ff33d899690789ea73f2708b6825f9>
> (88)
> > A Assets/Scripts/Tools/Utility/SetAnimatorParameter.cs.meta <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-48db24a79f00c2f7b2b4c3aa257c8360>
> (11)
> > A Assets/Scripts/Tools/Utility/SpawnOnContact.cs <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-c89da41e7a4f25fe51d776019988d4c9>
> (162)
> > A Assets/Scripts/Tools/Utility/SpawnOnContact.cs.meta <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-1d27ead4b14b252af3c7311de5fa5f54>
> (11)
> > A Assets/Scripts/Tools/Utility/SpawnOnShake.cs <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-62c20331226d8d618e6e1c350c0ff9d8>
> (65)
> > A Assets/Scripts/Tools/Utility/SpawnOnShake.cs.meta <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-a45f9d987b2c688a2a343bec62e2e21e>
> (11)
> > M Assets/Scripts/Wand.cs <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-88f88828271962312568f3c1b4d9d520>
> (11)
> > A Assets/Sound/ButaneSound.mp3 <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-b0d64686d838685499b7257a3944b9a0>
> (0)
> > A Assets/Sound/ButaneSound.mp3.meta <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-371837a1712eeefe10e6e279717bf9b5>
> (22)
> > A Assets/Sound/DefaultSpawn.mp3 <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-96d6ad3b47fb7c523ae9bc02df1c6a87>
> (0)
> > A Assets/Sound/DefaultSpawn.mp3.meta <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-d8b45eeac3c5ee261e9fff5c0e9060c8>
> (22)
> > A Assets/Sound/EthaneSound.mp3 <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-ef16bb932a5df900633bbcff0a41da83>
> (0)
> > A Assets/Sound/EthaneSound.mp3.meta <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-2c9eb10c2236ab326418e406a8e31c05>
> (22)
> > A Assets/Sound/FossilFuelsSpawn.mp3 <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-4d683439d0eeb9ffce0942e020155ca0>
> (0)
> > A Assets/Sound/FossilFuelsSpawn.mp3.meta <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-6521be085efe66b2dcbfdf89a9418e12>
> (22)
> > A Assets/Sound/HeptaneSound.mp3 <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-0c6557a9cecbfafbfff100e0f92d9774>
> (0)
> > A Assets/Sound/HeptaneSound.mp3.meta <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-1ff05c5f9e1886eda4650ad58f6fafbe>
> (22)
> > A Assets/Sound/HexaneSound.mp3 <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-6c2cd6893514ea2f5717c145a1a68e5b>
> (0)
> > A Assets/Sound/HexaneSound.mp3.meta <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-31bc6e37c809ee89cfbe86f25e69056b>
> (22)
> > A Assets/Sound/MethaneSound.mp3 <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-0223dee25d6165f500e70e12fa67a3f2>
> (0)
> > A Assets/Sound/MethaneSound.mp3.meta <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-0b29146abed75af1f662e6861ab6018a>
> (22)
> > A Assets/Sound/OctaneSound.mp3 <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-34b78f107b216ff1de224d43008f05d3>
> (0)
> > A Assets/Sound/OctaneSound.mp3.meta <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-ec9370c239ec4eb170a08a5adcd3e884>
> (22)
> > A Assets/Sound/PentaneSound.mp3 <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-ba09042521c6781facc5d912e94b85e6>
> (0)
> > A Assets/Sound/PentaneSound.mp3.meta <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-7907e1ab653db6c511cd88189e50c726>
> (22)
> > A Assets/Sound/PropaneSound.mp3 <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-7d6111a8a60284027fc613b0267f0f63>
> (0)
> > A Assets/Sound/PropaneSound.mp3.meta <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-c4c7c53a0556001dafb7cf3b71159585>
> (22)
> > A Assets/Sound/dino roar.mp3 <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-255122c139b6a7345ad8d642e4f21a1b>
> (0)
> > A Assets/Sound/dino roar.mp3.meta <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-63b623d5f1fb803ac92e169ec8ba04c2>
> (22)
> > A Assets/Sound/dino spawn.mp3 <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-7c9b24da97816bc0d91fa2cad7e673ec>
> (0)
> > A Assets/Sound/dino spawn.mp3.meta <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-87fb9d294091b3c16bb9bf3600e81e8a>
> (22)
> > A Assets/StreamingAssets/butanedata.json <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-7bea4901955193c35826abb424387b2f>
> (381)
> > A Assets/StreamingAssets/butanedata.json.meta <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-05083d4f0ffcea8f62f4d3b0144dd3b9>
> (7)
> > A Assets/StreamingAssets/ethanedata.json <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-9ae8ddb1249103970dde8b27dfb6fd0e>
> (322)
> > A Assets/StreamingAssets/ethanedata.json.meta <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-0ab4282bb833d242976e42e153bec18e>
> (7)
> > A Assets/StreamingAssets/heptanedata.json <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-15d06778f0b598cc782995f40fa6a995>
> (478)
> > A Assets/StreamingAssets/heptanedata.json.meta <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-7db429da67078414e5861836624c82f7>
> (7)
> > A Assets/StreamingAssets/hexanedata.json <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-0b21327cdf2e8d170e19747ee0a21e2f>
> (447)
> > A Assets/StreamingAssets/hexanedata.json.meta <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-a46d9f4a5e7da93630c79e7c628555b1>
> (7)
> > A Assets/StreamingAssets/methanedata.json <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-b47df1ac06c485a3f81e57ba7b1c3c56>
> (292)
> > A Assets/StreamingAssets/methanedata.json.meta <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-4a1e0f679b4c2a46d86c7091d9832272>
> (7)
> > A Assets/StreamingAssets/octanedata.json <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-89440417d9812c590b654a40dc460d9c>
> (537)
> > A Assets/StreamingAssets/octanedata.json.meta <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-540cf859493339be78c19a62276de0b6>
> (7)
> > A Assets/StreamingAssets/pentanedata.json <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-256cc8ac7939e99004701c9c9a65d249>
> (411)
> > A Assets/StreamingAssets/pentanedata.json.meta <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-4473d7103e9ed742e84af13376cf1418>
> (7)
> > A Assets/StreamingAssets/propanedata.json <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-cda1ffb2e2ac1694c654bc59a8cb306f>
> (348)
> > A Assets/StreamingAssets/propanedata.json.meta <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-39f7c527d5cf469119d428b65c021d9f>
> (7)
> > M ProjectSettings/EditorBuildSettings.asset <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-8f3c83eb79d0b4aa8cf581a168df971c>
> (18)
> > M ProjectSettings/ProjectSettings.asset <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-988fde71e6a4c917d4d31be70b435aa1>
> (6)
> > M ProjectSettings/TagManager.asset <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-c100fe01e4868a34c50c3f040849a2c6>
> (1)
> > M ProjectSettings/UnityConnectSettings.asset <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-3e2b55b7cb4b6b167c1e81679c6d94c9>
> (5)
> > A Rishi Astra - MolZooBuild.zip <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-fd4683e637696e8525e714b007172d4c>
> (0)
> > A build.zip <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-bd3f409385be2426c686b5e8eb288c66>
> (0)
> > M obj/Debug/Assembly-CSharp.csproj.FileListAbsolute.txt <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-84f93d4b7424e81423e9027b246070e4>
> (169)
> > M obj/Debug/Assembly-CSharp.dll <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-8f75c790a28d386e4316fcc8fb50be98>
> (0)
> > M obj/Debug/Assembly-CSharp.pdb <
> https://github.com/alanbrilliant/MolecularZoo/pull/1/files#diff-015c7d7ef0edc0f82de440447fb61084>
> (0)
> > Patch Links:
> >
> > https://github.com/alanbrilliant/MolecularZoo/pull/1.patch <
> https://github.com/alanbrilliant/MolecularZoo/pull/1.patch>
> > https://github.com/alanbrilliant/MolecularZoo/pull/1.diff <
> https://github.com/alanbrilliant/MolecularZoo/pull/1.diff>
> > —
> > You are receiving this because you are subscribed to this thread.
> > Reply to this email directly, view it on GitHub <
> https://github.com/alanbrilliant/MolecularZoo/pull/1>, or unsubscribe <
> https://github.com/notifications/unsubscribe-auth/AB6JDDJ6VSO64KONV5BYGOTRSMOC3ANCNFSM4NFOUAQA
> >.
> >
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/alanbrilliant/MolecularZoo/pull/1#issuecomment-631165102>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADBNTU6WX5KA57CCRDTMYCTRSMQ57ANCNFSM4NFOUAQA>
> .
>
-- 
*RAy ALTENBERG*

RishiAstra committed:
zip of latest build

RishiAstra committed:
Edits to Dino

Slowed down Dino speeds, molecules inheirit velocity, slowed down spawn rate, increased minimum velocity to spawn, etc.
